revolv/project/stats.py

```diff
@@ -78,7 +78,7 @@ class KilowattStatsAggregator(object):
     @classmethod
     def from_project(cls, project):
         """Instantage an aggregator from a Project models."""
-        return cls(project.impact_power)
+        return cls(project.impact_power, project.people_affected)
```

Comment:
I don't think KilowattStatsAggregator needs to know about this: instead of doing `project.stats.people_affected` in the template, we can just do `project.people_affected`